docs/resources/elasticsearch_enrich_policy.md

```diff
@@ -0,0 +1,89 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "elasticstack_elasticsearch_enrich_policy Resource - terraform-provider-elasticstack"
+subcategory: ""
```

Comment:
There's a shortcoming with the tf docs generator that doesn't allow setting the subcategory from code. 

Can we add a template ([example](https://github.com/elastic/terraform-provider-elasticstack/blob/main/templates/resources/elasticsearch_index.md.tmpl#L2))for this resource setting this field.